pkgs/development/python-modules/nbconvert/default.nix

```diff
@@ -47,6 +59,9 @@ buildPythonPackage rec {
     # Use mistune 2.x
     substituteInPlace setup.py \
         --replace "mistune>=0.8.1,<2" "mistune>=2.0.3,<3"
+
+    substituteInPlace share/jupyter/nbconvert/templates/lab/base.html.j2 \
+        --replace "{{ output.data['image/svg+xml'] | clean_html }}" "{{ output.data['image/svg+xml'].encode(\"utf-8\") | clean_html }}"
```

Comment:
Is this substitution something that upstream patched? (Maybe a comment could be nice to add?)

Reply:
yes, I added a comment